### ISSUE TITLE
[CF 1633] Execute not showing CD 

### ISSUE THREAD

**Issue by tmw-issue-import** [ascott18/TellMeWhen#1653]
<p>On my Fury warrior, have Execute selected as a Spell Cooldown option, with both Show Timer and Show Timer Text selected.&nbsp; Since 8.1, it no longer ever shows Execute as being on CD, even when under 20% (ie. when it doesn't have to rely on a Massacre proc).</p>
<p>&nbsp;</p>
<p>Digging into it, this might be a Blizzard issue.&nbsp; It looks like macros unconditionally try to cast the Arms version of Execute (163201) rather than the Fury version (5308), making Execute impossible to use in macros for Fury, and GetSpellCooldown("Execute") also seems to always return the Arms version's information.&nbsp; Even GetSpellCooldown(5308) doesn't seem to only sometimes return the properly date (ie. &lt;20%), and at other times return only the GCD length.</p> 
<br>

> Posted by CurseForge user <a href="https://wow.curseforge.com/members/Kaedys">Kaedys</a> | Imported from CurseForge issue <a href="https://wow.curseforge.com/projects/tellmewhen/issues/1633">#1633</a> | <a href="https://github.com/tmw-issue-import/dump/blob/master/html/1633.html">Raw</a>

**Comment by Jalopyy** [comment]
FWIW I simply made 2 auras to fix an issue I had with this, one that could only work when spec=arms and one with spec=fury.

**Comment by Alchymysts** [comment]
This issue with Execute - Fury spec, gets even weirder when you are in the Revendreth Covenant and get Condemn.
Execute used to work OK for me before that, before Shadowlands.
And it was sort of working OK until I acquired Condemn. With the use above 80% and below 20% it seems to have broken the show in the TMW bar properly.
And I do believe Blizzard also has a flaw in it's own actions with regards to the CD of 16 seconds.
Yes, I have it as a Reactive with Check Activation Border and Require Activation Border checked and 2 conditions set for the above 80% and below 20%. I have since dropped the conditions as it didn't seem to matter to Blizzard if they were met.
And to end, it seems there are about 30 Execute and/or Condemn IDs when looking directly.
